### PR TITLE
Bump wera2netcdf to 0.0.3

### PR DESCRIPTION
**Pull request by kwilcox**
* Support for empty ASCII files
* Support for a dynamic number of reporting stations

### CODE REVIEW

File: wera2netcdf/meta.yaml [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
 source:
-    fn: wera2netcdf-0.0.1.tar.gz
-    url: https://pypi.python.org/packages/source/w/wera2netcdf/wera2netcdf-0.0.1.tar.gz
-    md5: beb3bd457aebe6fa3d3b64cd2d0c0cbf
+    fn: wera2netcdf-0.0.3.tar.gz
+    url: https://pypi.python.org/packages/source/w/wera2netcdf/wera2netcdf-0.0.3.tar.gz
+    md5: 1970a907d222905c262619d420cc4f8f
 
 build:
     skip: True  # [win and py35]
```
Comment (ocefpaf) [inline wera2netcdf/meta.yaml]: @kwilcox can you remove this restriction so I can check if that is still true?

**Comment by kwilcox** [comment]
Yup, need a few

File: wera2netcdf/meta.yaml [hunk 2]
```diff
@@ -1,15 +1,14 @@
...
-    url: https://pypi.python.org/packages/source/w/wera2netcdf/wera2netcdf-0.0.1.tar.gz
-    md5: beb3bd457aebe6fa3d3b64cd2d0c0cbf
+    fn: wera2netcdf-0.0.3.tar.gz
+    url: https://pypi.python.org/packages/source/w/wera2netcdf/wera2netcdf-0.0.3.tar.gz
+    md5: 1970a907d222905c262619d420cc4f8f
 
 build:
-    skip: True  # [win and py35]
-    number: 0
+    number: 1
```
Comment (ocefpaf) [inline wera2netcdf/meta.yaml]: No need for this as we did not merge it yet.

**Comment by ocefpaf** [comment]
Because of https://github.com/ioos/conda-recipes/pull/764#issuecomment-185148737

Qe ended up with a release `0` that is preferred to the release `1` (see the WinPY35 issue comment above) , but conda will pick `1` up because it was wrongly uploaded.

I will fix this manually by deleting the release `1`.

**Pull request merged**